### PR TITLE
 iOS Issue #240 Add Priority to Tasks

### PR DESCRIPTION
**Pull request by cyndichin**
Issue #240 

## Description
* Added Priority section to the detail view for users to select priority associated with a task
* Added UI to show priority associated with a tasks
* Set default priority as medium

## Screenshots

https://github.com/WomenWhoCode/WWCodeMobile/assets/6743397/7bbe2f9f-f262-48dc-b397-43c496532e09

### CODE REVIEW

**Comment by cyndichin** [comment]
@clc80 @devanshimodha  This PR is ready for review, thank you!

File: coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift [hunk 1]
```diff
@@ -8,6 +8,12 @@
 import SwiftUI
 import SwiftData
 
+enum Priority: String, CaseIterable, Codable {
```
Comment (clc80) [inline coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift:11]: This is looking great. Can we move this enum into it's own file. The `DetailsView` is getting pretty long.

File: coding-projects/ios/TaskTracker/.DS_Store
Comment (clc80) [inline coding-projects/ios/TaskTracker/.DS_Store:1]: Let's not include this

**Review by clc80 — CHANGES_REQUESTED** [review]
Just some small changes.

File: coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift (hunk 1 above)
Comment (cyndichin) [reply coding-projects/ios/TaskTracker/TaskTracker/Details Screen/DetailsView.swift]: updated!

**Comment by cyndichin** [comment]
@clc80 thanks for your comments, they have been addressed, requested a re-review!

**Review by clc80 — APPROVED** [review]
Amazing job.

**Pull request merged**